modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminManagementToolbarDisplayContext.java

```diff
@@ -325,7 +326,12 @@ public List<DropdownItem> getFilterDropdownItems() {
 	public List<LabelItem> getFilterLabelItems() {
 		long fileEntryTypeId = _getFileEntryTypeId();
 
-		return LabelItemListBuilder.add(
+		LabelItemListBuilder.LabelItemListWrapper labelItemListWrapper =
+			new LabelItemListBuilder.LabelItemListWrapper();
+
+		_addFileExtensionFilterLabelItems(labelItemListWrapper);
+
+		labelItemListWrapper.add(
```

Comment:
Why don't chain these last `add` calls? Is SF complaining?

Reply:
Yes, SF was complaining 😓